integration-tests/features/steps/common.py

```diff
@@ -851,10 +863,27 @@ def check_outlier_probability(usage_outliers, package_name, threshold_value):
 def stack_analysis_check_outliers(context, component):
     json_data = context.response.json()
     threshold = context.outlier_probability_threshold
+    # log.info('Usage outlier threshold: %r' % threshold)
     path = "result/0/recommendations/usage_outliers"
     usage_outliers = get_value_using_path(json_data, path)
     check_outlier_probability(usage_outliers, component, threshold)
 
+@then('I should find that total {count} outliers are reported')
+def check_outlier_count(context, count=2):
+    json_data = context.response.json()
+    path = "result/0/recommendations/usage_outliers"
+    usage_outliers = get_value_using_path(json_data, path)
+    assert len(usage_outliers) == int(count)
+
+@then('I should find that valid outliers are reported')
+def check_outlier_validity(context):
+    json_data = context.response.json()
+    threshold = 0.9
```

Comment:
Unfortunately, context.outlier_probability_threshold did not work for us. And from what I've heard from Tuhin, the threshold approach is going to change. So, I think we are safe here.